app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,13 @@ def destroy
       end
 
       def find_parent_config_template
```

Comment:
Actually ```@config_template``` is used and is required by ```:index,  :create``` (failing without it.)
I tried to removed it, as I thought it will be through ```nested_obj``` but those actions require it.